content/en/docs/collector/configuration.md

```diff
@@ -247,28 +247,31 @@ receivers:
   zipkin:
 ```
 
+> For detailed receiver configuration, see the
+> [receiver README](https://github.com/open-telemetry/opentelemetry-collector/blob/main/receiver/README.md).
+
 ## Processors <img width="35" class="img-initial" src="/img/logos/32x32/Processors.svg"> {#processors}
 
-Processors are run on data between being received and being exported. Processors
-are optional though
-[some are recommended](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor#recommended-processors).
+Processors take the data collected by receivers and modify or transform it
```

Comment:
Not seeing a note in here about the ordering of processor matters, it's possible i missed it though

Reply:
Added in line 258.